Polyhedron/demo/Polyhedron/Plugins/PMP/Mean_curvature_flow_skeleton_plugin.cpp

```diff
@@ -28,6 +28,7 @@
 #include <CGAL/extract_mean_curvature_flow_skeleton.h>
 #include <CGAL/iterator.h>
 #include <CGAL/Polygon_mesh_processing/connected_components.h>
+#include <CGAL/Polyhedron_items_with_id_3.h>
```

Comment:
Do you really need that?

Reply:
I guess yes, but note that it is in the HDS package.

Reply:
My point is that the demo uses surface mesh so why would the demo need that

Reply:
It was needed to provide the generic version for [this specialization](https://github.com/CGAL/cgal/pull/6547/files#diff-1e1f7269cc6361055ad7a3c053fe02225d77c9506833fceab57ab9fdf211f81bL43)  which I just removed as it is a no-op.

Reply:
Cleaned with [eea9cf7](https://github.com/CGAL/cgal/pull/6547/commits/eea9cf7ac4e05ad6f49d94c0f2a7ca1864ffc74d)